pytarallo/Tarallo.py

```diff
@@ -1,12 +1,16 @@
 import json
 import urllib.parse
-import requests
+import pytarallo.Errors as Errors
```

Comment:
This _should_ be equivalent:
```suggestion
import Errors
```